grails5

niravassar opened the pull request.

Commit:
grails5

sdelamo commented on complete/build.gradle:
```diff
@@ -5,9 +5,8 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails version

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,7 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
\ No newline at end of file
+grailsVersion=5.0.0
```
grails version should be 5.0.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,7 @@
-gormVersion=7.0.2.RELEASE
\ No newline at end of file
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be GORM 7.1.0

sdelamo commented on initial/build.gradle:
```diff
@@ -5,9 +5,8 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails version

sdelamo commented on initial/build.gradle:
```diff
@@ -18,12 +17,12 @@ apply plugin:"eclipse"
 apply plugin:"org.grails.grails-gsp"
 
 repositories {
     mavenLocal()
```
remove mavenLocal

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
\ No newline at end of file
+grailsVersion=5.0.0
```
grails version should be 5.0.1

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-gormVersion=7.0.2.RELEASE
\ No newline at end of file
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be GORM 7.1.0

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
